redbot/cogs/trivia/data/lists/minecraft.yml

```
@@ -0,0 +1,114 @@
+AUTHOR: ImajinDevon
+
+"The phantom was added in which update?":
+ - "1.13"
+"Which update introduced a new style combat into the game?":
+ - "1.9"
+'What edition of Minecraft is usually prefixed with the word "bug"?':
+ - "Bedrock"
+"Who created Minecraft?":
+ - "Notch"
+"Who bought Mojang in 2014?":
+ - "Microsoft"
+"Mojang was bought for (...) billion dollars:":
+ - "2.5"
+"If you name a sheep this, it will turn rainbow:":
+ - "jeb_"
+"What supercharges creepers?":
+ - "lightning"
+"What are creepers scared of?":
+ - "cats"
+"To kill an iron golem without getting hit, you must build how many blocks up?":
+ - "3"
+"What is an enderman's weakness?":
+ - "water"
+"What block lights up when you step on it?":
+ - "redstone ore"
+"If you trip this, redstone power will be emitted:":
+ - "tripwire hook"
+"Upon opening this container, redstone power will be emitted:":
+ - "trap chest"
+"This myth has commonly passed around the Minecraft community for years:":
+ - "herobrine"
+"To obtain the dragon's egg, you must have it fall onto which item?":
+ - "torch"
+"What mob dances when music played?":
+ - "parrot"
+ - "parrots"
+"What is the name of the most popular arcade and minigame server?":
+ - "hypixel"
+"What is the default port of a Minecraft server?":
+ - "25565"
+"This block was added in 1.19, and emits small amounts of light:":
+ - "glow lichen"
+"Which item besides cactus, when planted, pricks you when stepped on?":
+ - "sweet berries"
+"What item is needed to make green dye?":
+ - "cactus"
+"What ore is only found in the nether, and grants decent amounts of XP?":
+ - "quartz"
+"What ore drop is needed to enchant items in an enchantment table? (full name)":
+ - "lapis lazuli"
+"An enderman gets angry if you look at its:":
+ - "eyes"
+"What mob infects stone?":
+ - "silverfish"
+"This mob makes you fly when you are hit by its projectile:":
+ - "shulker"
+"This container allows you to access storage across any dimension, and its contents are unique to the player who opened it:":
+ - "enderchest"
+ - "ender chest"
+"Which item, when placed in anvil allows you to repair damaged/broken elytras?":
+ - "phantom membrane"
+"This ORE and its successors do not burn in lava:":
+ - "ancient debris"
+"To place a chest next to another without joining them, you must do what type of click?":
+ - "shift click"
+ - "shiftclick"
+ - "shift rightclick"
+ - "shift right click"
```

Not every uses the same key binds as you, so you should not refer to the keys that way.

---

This behavior is also not limited to placing chests, but the question probably becomes too confusing to reference all blocks with inventories
```suggestion
"To place a chest next to another without opening it, you must do what while clicking?":
 - "sneak"
```